tensorboard/dataclass_compat.py

```diff
@@ -72,11 +76,15 @@ def _migrate_graph_event(old_event, experimental_filter_graph=False):
 
     # TODO(@davidsoergel): Move this stopgap to a more appropriate place.
     if experimental_filter_graph:
-        graph_def = graph_pb2.GraphDef().FromString(graph_bytes)
-        # Use the default filter parameters:
-        # limit_attr_size=1024, large_attrs_key="_too_large_attrs"
-        process_graph.prepare_graph_for_ui(graph_def)
-        graph_bytes = graph_def.SerializeToString()
+        try:
+            graph_def = graph_pb2.GraphDef().FromString(graph_bytes)
+            # Use the default filter parameters:
+            # limit_attr_size=1024, large_attrs_key="_too_large_attrs"
+            process_graph.prepare_graph_for_ui(graph_def)
+            graph_bytes = graph_def.SerializeToString()
```

Comment:
You can move these two lines outside the try scope, IIUC.

Reply:
Done.